quickvision/datasets/classification.py

```diff
@@ -40,26 +40,31 @@ class CSVDataset(Dataset):
     """
```

Comment:
Can you rename this with `CamelCase` as `CSVImageidDataset` ? or something similar ?

Reply:
I am renaming it to `CSVSingleLabelDataset`.